network-api/networkapi/wagtailpages/tests/libraries/rcc/utils.py

```diff
@@ -20,29 +20,36 @@ def translate_detail_page(detail_page, locale):
     # Requires previous tree synchronization
     trans_detail_page = detail_page.get_translation(locale)
 
-    for research_author_trans in trans_detail_page.research_authors.all():
+    for rcc_author_trans in trans_detail_page.rcc_authors.all():
         # The through model is already for the new locale after the tree sync,
         # but the related model is not.
-        author_profile_orig = research_author_trans.author_profile
+        author_profile_orig = rcc_author_trans.author_profile
         author_profile_trans = author_profile_orig.copy_for_translation(locale)
         author_profile_trans.save()
-        research_author_trans.author_profile = author_profile_trans
-        research_author_trans.save()
+        rcc_author_trans.author_profile = author_profile_trans
+        rcc_author_trans.save()
+
+    for related_ct_trans in trans_detail_page.related_content_types.all():
+        ct_orig = related_ct_trans.content_type
+        ct_trans = ct_orig.copy_for_translation(locale)
+        ct_trans.save()
+        related_ct_trans.content_type = ct_trans
+        related_ct_trans.save()
+
+    for related_ca_trans in trans_detail_page.related_curricular_areas.all():
+        ca_orig = related_ca_trans.curricular_area
+        ca_trans = ca_orig.copy_for_translation(locale)
+        ca_trans.save()
+        related_ca_trans.content_type = ca_trans
+        related_ca_trans.save()
```

Comment:
**Suggestion**

I think naming variable a bit cleary would improve the readability of this
- related_ct_trans
- ct_trans
- ct_orig
- related_ca_trans
- ca_trans
- ca_orig

What's `ct` and what's `ca`?

Reply:
`ca` is `curricular_area` and `ct` is `content_type`. If it improves readability I can get those back to the long form